### PR TITLE
Replace functional.sigmoid with torch.sigmoid in RPN

### PR DESCRIPTION
**Pull request by fmassa**
Removes deprecation warning introduced in #3205 
```
/Users/fmassa/anaconda3/lib/python3.6/site-packages/torch/nn/functional.py:1709: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
```

### CODE REVIEW

**Review by datumbox — APPROVED** [review]
LGTM!
